activemodel/lib/active_model/locale/en.yml

```diff
@@ -14,6 +14,7 @@ en:
       empty: "can't be empty"
       blank: "can't be blank"
       present: "must be blank"
+      required: "must exist"
```

Comment:
If this is actually going in, both the `required` locale change and the changelog should be part of active record.